.readthedocs.yml

```diff
@@ -9,14 +12,9 @@ build:
     - libgl1
     - xvfb
   jobs:
-    pre_build:
-      # Generate the Sphinx configuration for this Jupyter Book so it builds.
-      - "jupyter-book config sphinx book/"
-
-conda:
-  environment: environment.yml
+    post_build:
+      # this may need the --execute flag to run the notebooks
+      - cd book && jupyter book build --html
+      - cp -r book/_build/html/* $READTHEDOCS_OUTPUT/html/
```

Comment:
```suggestion
    build:
      html:
        # this may need the --execute flag to run the notebooks
        - cd book && jupyter book build --html
    post_build:
      - cp -r book/_build/html/* $READTHEDOCS_OUTPUT/html/
```

Reply:
I think the error you're getting is because RTD is trying to build with Sphinx. You'll need to override the `build` job to use Jupyter Book instead: https://docs.readthedocs.com/platform/stable/build-customization.html